### PR TITLE
Test executables in bin

### PR DESCRIPTION
**Pull request by cbguder**
This would help us catch regressions such as the one fixed by #321. The smoke tests are fairly basic right now, but it's a start.

### CODE REVIEW

File: bin/grouper-fe [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 from grouper.plugin import get_plugins, load_plugins
-from grouper.settings import default_settings_path
-from grouper.setup import parse_args, setup_logging
-from grouper.util import get_loglevel, get_database_url
+from grouper.setup import build_arg_parser, setup_logging
+from grouper.util import get_database_url
 
 
-def get_application(db_session_cls, settings, sentry_client, deployment_name):
-    # type: (Session, Settings, SentryProxy, str) -> Application
```
Comment (herb) [inline bin/grouper-fe]: can you talk about why you removed this parameter?

Comment (herb) [reply bin/grouper-fe]: nvm. it is indeed unused.

File: bin/grouper-api [hunk 2]
```diff
@@ -1,31 +1,30 @@
 #!/usr/bin/env python
 
 import argparse
-from contextlib import closing
 import logging
 import os
 import sys
+from contextlib import closing
```
Comment (herb) [inline bin/grouper-api]: nit: we started sorting alphabetically in more recent diffs. so 

```
from contextlib import closing
import logging
import os		  import os
import sys		  import sys
```

Comment (cbguder) [reply bin/grouper-api]: ah, fair enough. I'll reorder accordingly.

**Pull request merged**